frontend/app_flowy/packages/appflowy_editor_plugins/lib/src/callout/callout_node_widget.dart

```diff
@@ -0,0 +1,279 @@
+import 'package:appflowy_editor/appflowy_editor.dart';
+import 'package:appflowy_editor_plugins/src/emoji_picker/emoji_menu_item.dart';
+import 'package:appflowy_popover/appflowy_popover.dart';
+import 'package:flowy_infra/l10n.dart';
+import 'package:flowy_infra/theme_extension.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flowy_infra_ui/style_widget/button.dart';
+import 'package:flowy_infra_ui/style_widget/color_picker.dart';
+import 'package:flowy_infra_ui/style_widget/icon_button.dart';
+import 'package:flutter/material.dart';
+
+const String kCalloutType = 'callout';
```

Comment:
This type should be `text/callout`.

Because callout is similar to reference, just a different presentation of TextNode. if callout is used as a separate type, it cann't be converted to other styles of text.

For example
text/callout and text/quote are converted to each other, while callout cann't be converted to text/quote

The meaning of the separator is, before the separator as type, after the separator as subtype, indicating that subtypes can be converted to each other.

Reply:
I think there could be some use cases, where someone wants to put different nodes inside a callout.
e.g.: formula with text.
![image](https://user-images.githubusercontent.com/8122139/215275866-2a774b1b-512d-40e3-a881-21475f2d6cfa.png)
That is why I made callout a separate type, but if you prefer `text/callout`, I'm happy to change it.

Reply:
Good insight. If we want to put another node into the callout, then using callout would be better.